Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -543,7 +550,7 @@ extension SwiftLanguageService {
       let replacement: String
     }
 
-    guard let (preEditSnapshot, postEditSnapshot, edits) = self.documentManager.edit(notification) else {
+    guard let (preEditSnapshot, postEditSnapshot, edits) = try? self.documentManager.edit(notification) else {
```

Comment:
Similar here, could you use `orLog` so we log an error if the document manager is `nil`?

```suggestion
    guard let (preEditSnapshot, postEditSnapshot, edits) = orLog("Getting document manager", { try self.documentManager.edit(notification) } else {
```